demoapp/composeApp/src/commonMain/kotlin/at/asitplus/cryptotest/App.kt

```diff
@@ -112,22 +112,29 @@ internal fun App() {
             X509SignatureAlgorithm.RS1,
             X509SignatureAlgorithm.RS256,
             X509SignatureAlgorithm.RS384,
-            X509SignatureAlgorithm.RS512)
-        var keyAlgorithm by remember { mutableStateOf<SpecializedSignatureAlgorithm>(X509SignatureAlgorithm.ES256) }
+            X509SignatureAlgorithm.RS512
+        )
+        var keyAlgorithm by remember {
+            mutableStateOf<SpecializedSignatureAlgorithm>(
+                X509SignatureAlgorithm.ES256
+            )
+        }
         var inputData by remember { mutableStateOf("Foo") }
         var currentSigner by remember { mutableStateOf<KmmResult<Signer>?>(null) }
         val currentKey by getter { currentSigner?.mapCatching(Signer::publicKey) }
         val currentKeyStr by getter {
             currentKey?.fold(onSuccess = {
                 it.toString()
             },
-            onFailure = {
-                Napier.e("Key failed", it)
-                "${it::class.simpleName ?: "<unnamed>"}: ${it.message}"
-            }) ?: "<none>"
+                onFailure = {
+                    Napier.e("Key failed", it)
+                    "${it::class.simpleName ?: "<unnamed>"}: ${it.message}"
+                }) ?: "<none>"
         }
         val currentAttestation by getter { (currentSigner?.getOrNull() as? Signer.Attestable<*>)?.attestation }
-        val currentAttestationStr by getter { currentAttestation?.jsonEncoded ?: "" }
+        val currentAttestationStr by getter {
+            currentAttestation?.jsonEncoded?.also { Napier.d { it } } ?: ""
+        }
```

Comment:
debug leftover?

Reply:
no, not really. helps to extract it to verify

Reply:
make it a sensible log entry then, e.g., `"Current attestation:\n${it}"`